README.md

```diff
@@ -18,35 +18,12 @@ taking our pull requests, implementing feature requests and fixing bugs! See
 [Notes on CodeMirror](https://github.com/adobe/brackets/wiki/Notes-on-CodeMirror)
 for info on how we're using CodeMirror.
 
-#How to setup Nimble with MakeDrive server
+# How to setup Bramble (Brackets) in your local machine
 
-## Setup Webmaker Login Server
-
-You need to have Webmaker Login for Authentication to connect to MakeDrive WebSocket Server.  
-
-Make sure you have forked and cloned [Webmaker Login](https://github.com/mozilla/login.webmaker.org) then run the following steps:  
-
-Install npm modules:
-```
-$ npm install
-```
-Use the default configuration:
-```
-$ cp env.sample .env
-```
-Run the server:
-```
-$ node app.js
-```
-
-You will now have Webmaker Login server running on port `3000` you can test by visiting [http://localhost:3000/account](http://localhost:3000/account).  Also, make sure you create an account.
-
-## Setup Nimble (Brackets) in your lcoal machine
-
-Step 01: Make sure you have forked and clone [Nimble](https://github.com/mozilla/nimble).
+Step 01: Make sure you fork and clone [Bramble](https://github.com/humphd/brackets).
 
 ```
-$ git clone https://github.com/[yourusername]/nimble --recursive
+$ git clone https://github.com/[yourusername]/brackets
```

Comment:
Careful, you removed instructions on forking a repo so this line is no longer correct. Also, the `--recursive` option is important since it clones submodules as well.